lib/manageiq/api/common/graphql/generator.rb

```diff
@@ -67,6 +67,16 @@ def self.collection_field_resolvers(schema_overlay, collection)
             field_resolvers
           end
 
+          def self.collection_schema_overlay(schema_overlay, collection)
+            collection_schema_overlay = {}
+            schema_overlay.keys.each do |collection_regex|
```

Comment:
minor simplification, but this can be 

```ruby
schema_overlay.keys.each_with_object({}) do |collection_regex, collection_schema_overlay|
```

and then you don't need to add the extra explicit `collection_schema_overlay` return at the end